packages/web-twig/src/Resources/components/Breadcrumbs/index.twig

```diff
@@ -23,7 +23,9 @@
                     </li>
                 {% endif %}
                 <li class="d-none d-tablet-flex">
-                    <Icon name="chevron-right" />
+                    {% if loop.index != 1 %}
```

Comment:
If you would you `index0` then the condition could be basically the same as in JS. But 🤷.